test/Microsoft.SqlTools.ServiceLayer.Test.Common/TestCredentialService.cs

```diff
@@ -0,0 +1,98 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System.Collections.Generic;
+using System.Globalization;
+using Microsoft.SqlTools.ServiceLayer.Credentials;
+using Microsoft.SqlTools.ServiceLayer.Credentials.Contracts;
+
+namespace Microsoft.SqlTools.ServiceLayer.Test.Common
+{
+    public class TestCredentialService
+    {
+        private CredentialService _credentialService = TestServiceProvider.Instance.CredentialService;
```

Comment:
it'd be nice if we had a stylecop running to ensure we all align on this stuff, since when we port code over it's a pain to keep in sync. Just a thought :)